exercises/raindrops/description.md

```diff
@@ -1,16 +1,14 @@
-Convert a number to a string, the contents of which depend on the number's factors.
+In this variant of the children's game [Fizz Buzz](https://en.wikipedia.org/wiki/Fizz_buzz) -- which has often been used as a screening exercise in technical job interviews -- your task is to convert an number into a string that contains raindrop sounds corresponding to certain potential factors. A factor is a number that evenly divides into another number, leaving no remainder. The simplest way to test if a one number is a factor of another is to use the [modulo operation](https://en.wikipedia.org/wiki/Modulo_operation).
```

Comment:
```suggestion
Your task is to convert an number into a string that contains raindrop sounds corresponding to certain potential factors. A factor is a number that evenly divides into another number, leaving no remainder. The simplest way to test if a one number is a factor of another is to use the [modulo operation](https://en.wikipedia.org/wiki/Modulo_operation).
```

The FizzBuzz reference is accurate, but it's not the most important thing about the exercise, so I don't want this to be the opening sentence.

I love the more specific reference though. Can we add it to the YAML file instead as the source? That would bring it into the final README.

https://github.com/exercism/problem-specifications/blob/6823334bbbef8e6193c09483fb3e5865f5f87a2b/exercises/raindrops/metadata.yml#L3

Reply:
@kytrinyx happy to move it to the metadata.yml, but would that mean updating the source_url to the wikipedia link, or leaving in jumpstartlab.com? From the WP link FizzBuzz has been a children's game for teaching math since at least 2002, and its use as a programming interview question seems to have started in 2007 (https://imranontech.com/2007/01/24/using-fizzbuzz-to-find-developers-who-grok-coding/).

Reply:
@kytrinyx pushed an update of the metadata.yml ... are those changes sufficient?

Reply:
Yeah, I like the new `source`! Much better.